### PR TITLE
feat(entity): create entity for 2 user and user authen, PEFIMA-11

### PR DESCRIPTION
**Pull request by lntrTin0212**

### CODE REVIEW

**Comment by lntrTin0212** [comment]
@PhucPhan11 Hi, can you look over this PR and give me the feedback? I will self test it also

File: src/main/java/com/ceu/Entity/InternalUserAuthentication.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity
+@Table(name = "internal-user-authentication")
```
Comment (PhucPhan11) [inline src/main/java/com/ceu/Entity/InternalUserAuthentication.java:20]: from my point of view, the name of table should be ABC_DEF or abc_def, you can use "_" instead of "-"

File: src/main/java/com/ceu/Entity/User.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+
+@Entity
+@Table(name = "user-account")
```
Comment (PhucPhan11) [inline src/main/java/com/ceu/Entity/User.java:23]: same with internal user

File: src/main/java/com/ceu/model/Gender.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.ceu.model;
+
+public enum Gender {
+    MALE,
```
Comment (PhucPhan11) [inline src/main/java/com/ceu/model/Gender.java:4]: I think the enum class should be in the same format

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1,5 +1 @@
-quarkus.datasource.db-kind=postgresql
-quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/postgres
-quarkus.datasource.username=docker
-quarkus.datasource.password=docker
-quarkus.hibernate-orm.database.generation=update
```
Comment (PhucPhan11) [inline src/main/resources/application.properties:5]: why did you remove those properties, Does that affect the connection between service and database?

**Comment by PhucPhan11** [comment]
@lntrTin0212 could you please take a look at my review

File: src/main/java/com/ceu/Entity/InternalUserAuthentication.java (hunk 1 above)
Comment (PhucPhan11) [reply src/main/java/com/ceu/Entity/InternalUserAuthentication.java:20]: @lntrTin0212

File: src/main/java/com/ceu/Entity/User.java (hunk 2 above)
Comment (PhucPhan11) [reply src/main/java/com/ceu/Entity/User.java:23]: @lntrTin0212

File: src/main/java/com/ceu/model/Gender.java (hunk 3 above)
Comment (PhucPhan11) [reply src/main/java/com/ceu/model/Gender.java:4]: @lntrTin0212

File: src/main/resources/application.properties (hunk 4 above)
Comment (PhucPhan11) [reply src/main/resources/application.properties:5]: @lntrTin0212

File: src/main/java/com/ceu/Entity/User.java (hunk 2 above)
Comment (lntrTin0212) [inline src/main/java/com/ceu/Entity/User.java:23]: Sorry, typo @PhucPhan11

File: src/main/resources/application.properties (hunk 4 above)
Comment (lntrTin0212) [reply src/main/resources/application.properties:5]: I removed this properties because its not in my scope, currently i'm using dev service from quarkus to test entity and flyway, you can add it later when implementing feature related to config and connect database
